### DISCUSSION TITLE
Picture

### DISCUSSION THREAD

**Discussion by rafikApp** [Budibase/budibase#6424] [Help]
I have this problem when I add a photo and click on save

![p](https://user-images.githubusercontent.com/106697208/175010366-87f04ecc-d118-4540-be30-65fb03777788.PNG)

**Comment by melohagan** [comment]
Hey @rafikApp 

Could you share any errors that you see in the console/networks tabs please?

**Comment by rafikApp** [comment]
![1](https://user-images.githubusercontent.com/106697208/175084301-c59cf2f0-e193-44a1-858c-51bb8fbcd2a9.PNG)

**Reply by melohagan** [reply]
Thank you, and if you screenshot the `.../rows` response from the network tab - is there more detail in there?

**Comment by rafikApp** [comment]
Hi @melohagan 

![2](https://user-images.githubusercontent.com/106697208/175254628-74cab164-6e4e-4bc0-bf20-6f1050f62e64.PNG)

**Reply by melohagan** [reply]
Then if you click on rows, a panel will appear on the right hand-side. Then click on response and take a screenshot of that.

**Comment by rafikApp** [comment]
![3](https://user-images.githubusercontent.com/106697208/175262818-e90d9410-2113-4e83-b30b-d36c34d852ab.PNG)



![4](https://user-images.githubusercontent.com/106697208/175262840-8a6982a5-9c7b-48f2-9467-43a46568da42.PNG)

**Reply by melohagan** [reply]
Hey @rafikApp If you click on the Response tab of the second screenshot, could you share the error message that's in there please?

**Reply by rafikApp** [reply]
Hi @melohagan 
{"message":"Bad Request","status":400,"validationErrors":{"photo":["must be of type array"]}}

**Reply by rafikApp** [reply]
I think I found the problem, when I use a form generated by budibase it works fine, but when I use a form I created it doesn't work

**Reply by melohagan** [reply]
Thank you @rafikApp 
This issue is related: https://github.com/Budibase/budibase/issues/5697

In short, you can use a JavaScript binding instead of handlebars and that should work.

**Reply by rafikApp** [reply]
ok Thank you @melohagan

**Comment by rafikApp** [comment]
Any solution ?
